### PR TITLE
integrated thermal test and plotting features for the test results

### PR DESCRIPTION
**Pull request by Acuda**
related to https://github.com/mojin-robotics/cob4/issues/1125

### CODE REVIEW

File: cob_devices_test/README.md [hunk 1]
```diff
@@ -72,3 +72,30 @@ Test Speakers with **Test Speakers** within the Sound Settings (check if left an
 
 ### Touchscreen:
 Touch the display of the robot
+
+## Intel NUC
```
Comment (fmessmer) [inline cob_devices_test/README.md]: should we move this to https://github.com/mojin-robotics/cob_hardware_test/blob/indigo_dev/README_nuc.md?
the scripts could stay in `cob_test_rigs` - I don't know a more fitting location

Comment (fmessmer) [inline cob_devices_test/README.md]: with this Doku and @Acuda's scrpts, do we still need #34 (by @benmaidel)
please combine the two PRs into a single one that contains all relevant scripts, docu and hints in one place!

Comment (floweisshardt) [reply cob_devices_test/README.md]: is fine here

Comment (floweisshardt) [reply cob_devices_test/README.md]: #34 is closed

**Pull request merged**